translations/base-de.yaml

```diff
@@ -177,7 +177,7 @@ dialogs:
         title: Speicherstand importiert
         text: Dein Speicherstand wurde erfolgreich importiert.
     gameLoadFailure:
-        title: Der Speicherstand ist kaputt
+        title: Der Speicherstand ist kaputt oder fehlerhaft
```

Comment:
Das ist meiner Meinung nach doppelt gemoppelt, da es sich um den Titel des Fensters handelt (zu lang). Du kannst die zusätzliche Info gerne in die Zeile drunter packen